test/unit-test/w3cunittest.cpp

```diff
@@ -1135,7 +1135,7 @@ BOOST_AUTO_TEST_CASE(process_query_set_get_simple)
     "action": "get",
     "path": "Vehicle.OBD.EngineSpeed",
     "requestId": "8756",
-    "value": 2345
```

Comment:
@SebastianSchildt why sometime speed is flot sometime int32

Reply:
Is "I have no idea" a valid answer here? Vehicle.OBD.Speed is defined as float  and Vehicle.Speed as int32  in the VSS model (maybe be a question to take to Genivi :) ) . I am not sure, if maybe the rendering via our jsonlib changed somehow for floats with no decimal places... And I am not sure how clever our impl is when writing an int to a float data item or vice versa.....

Reply:
This:

<img width="501" alt="image" src="https://user-images.githubusercontent.com/3707124/98272319-40a1da00-1f91-11eb-822a-eb6b031f25bc.png">

doesn't inspire confidence...